core/ecschema-editing/src/Editing/ECClasses.ts

```diff
@@ -283,6 +283,57 @@ export class ECClasses {
     return {};
   }
 
+  /**
+   * Renames the property on the specified class. The rename will fail if the new
+   * name causes a conflict with a base or derived property. All derived classes
+   * containing a property override will be renamed, as well.
+   * @param classKey  The SchemaItemKey identifying the class.
+   * @param existingPropertyName The name of the property.
+   * @param newPropertyName The new property name.
+   */
+  public async setPropertyName(classKey: SchemaItemKey, existingPropertyName: string, newPropertyName: string): Promise<PropertyEditResults> {
+    const newName = new ECName(newPropertyName);
+
+    let mutableClass: MutableClass;
+    try {
+      mutableClass = await this.getClass(classKey);
+    } catch (e: any) {
+      return { errorMessage: e.message };
+    }
+
+    // Does this suffice for not allowing property overrides to be renamed?
```

Comment:
what is this comment about?

Reply:
Is was a question I had for you, but I can remove it. I think we are good now. I removed it.